src/scripts/updateVideoDateTime.ts

```diff
@@ -0,0 +1,35 @@
+import { readFileSync } from 'fs';
+import db from '../db/index';
+import { exit } from 'process';
+
+export function readJson(path: string) {
+  const data = JSON.parse(readFileSync(path, 'utf-8'));
+  return data;
+}
+
+async function updateVideoDateTime(jsonPath: string) {
+  const data = readJson(jsonPath);
+  try {
+    await db.$transaction(async (tx) => {
+      for (const { id, date, timestamp } of data) {
```

Comment:
The data in [cohort2.json](https://github.com/code100x/cms/blob/491ab2862eb834ca1433b764cb709a44705c19ea/src/scripts/dateAndTimestampData/cohort2.json) file does not have any `timestamp` key, it is `duration`.